test-files/rexster/rexster-onlyclasses/src/main/java/org/jboss/windup/graph/rexster/RexsterInitializer.java

```diff
@@ -0,0 +1,291 @@
+package org.jboss.windup.graph.rexster;
```

Comment:
Oh, this is in test-files as the source for your test. nvm, I understand now. :)